ptsa/data/__init__.py

```diff
@@ -1,7 +0,0 @@
-# FIXME: all of this is old PTSA stuff that should be removed
```

Comment:
This file should still exist even if empty.

Reply:
This file does still exist, but Github doesn't format the diff differently for empty files vs removed files